org.kosmospredanie.Yakblt.json

```diff
@@ -0,0 +1,63 @@
+{
+	"app-id": "org.kosmospredanie.Yakblt",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "19.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "org.kosmospredanie.Yakblt",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=x11",
+		"--device=dri",
+		"--socket=pulseaudio",
+		"--filesystem=~/.yakblt-scores.txt"
```

Comment:
Any reason for needing access to this?

Reply:
The program saves game scores to the file.

Reply:
You should $XDG_DIRS for that. https://docs.flatpak.org/en/latest/conventions.html#xdg-base-directories